setup.py

```diff
@@ -143,8 +144,9 @@
     },
     project_urls={
         "Documentation": "https://www.thainlp.org/pythainlp/docs/dev/",
```

Comment:
Should we link the document to the latest (stable) version?

Reply:
Agree. Will change accordingly.